lib/active_model/serializer.rb

```diff
@@ -50,6 +50,10 @@ def self.type(type)
       self._type = type
     end
 
+    def self.links(&block)
+      self._links_block = block
+    end
```

Comment:
My gut is to avoid long-lived blocks, as there may be a lot of these, and the memory would never be released